src/Symfony/Bridge/Twig/Extension/DumpExtension.php

```diff
@@ -23,10 +24,12 @@
 class DumpExtension extends \Twig_Extension
 {
     private $cloner;
+    private $dumper;
 
-    public function __construct(ClonerInterface $cloner)
+    public function __construct(ClonerInterface $cloner, DataDumperInterface $dumper = null)
```

Comment:
OK, nevermimd this idea, constructor injection looks fine :-) I was confused with something else related to my PR.